Components/Text Utilities/text utils.css

```diff
@@ -0,0 +1,17 @@
+@import "../../Colors/colors.css";
+
+.heading {
```

Comment:
Headings can be of different sizes try to implement more classes for those sizes as well moreover only defining fonts size is not enough try incorporating more properties.

Reply:
On it...